common/v2/components/Tabs.tsx

```diff
@@ -24,16 +24,15 @@ interface TabWrapperProps {
 const TabWrapper = styled.div<TabWrapperProps>`
   ${props =>
     props.selected &&
-    `background-color: ${BRIGHT_SKY_BLUE};
-     color: ${WHITE};`}
-  padding: 12px 20px;
+    `background-color: ${BLUE_BRIGHT};
+     color: ${WHITE};`} padding: 12px 20px;
```

Comment:
missing new-line

Reply:
The linter thing did this as well. I'll ping you on Slack to see if that's a Taylor+her linter thing or a global linter thing?